frontend/build/profiles/integration/annotation-tool-configuration.js

```diff
@@ -151,20 +153,194 @@ define(["jquery",
+
+                        // Get comments by user
+                        if (!annotation.areCommentsLoaded()) {
+                            annotation.fetchComments();
+                        }
+
+                        _.each(annotation.attributes.comments.models, function (comment) {
+                            addCommentLine(line, comment);
+                            
+                            if(comment.replies.length > 0) {
+                                comment_replies(line, comment.replies.models)
+                            }
+                        });
+
+                    });  
                 });
-                parameters.append("freetext", freeText);
-                window.location.href =
-                    "../extended-annotations/videos/" +
-                    video.id +
-                    "/export.csv?" +
-                    parameters;
+
+                return bookData;
+
+                function addResourceHeaders(header, presuffix = "") {
+                    let prefix = ""
+                    let suffix = ""
+                    if(presuffix) {
+                        prefix = presuffix + " "
+                        suffix = " of " + presuffix
+                    }
+                    header.push(util.capitalize(prefix + "ID"));
+                    header.push(util.capitalize(prefix + "Creation date"));
+                    header.push(util.capitalize("Last update" + suffix));
+                    header.push(util.capitalize(prefix + "Author nickname"));
+                    header.push(util.capitalize(prefix + "Author mail"));
+                }
+
+                function addResource(line, resource) {
+                    line.push(resource.id);
+                    line.push(resource.attributes.created_at.toISOString());
+                    line.push(resource.attributes.updated_at.toISOString());
+                    line.push(resource.attributes.created_by_nickname);
+                    line.push(resource.attributes.created_by_email);
+                }
+
+                function addCommentLine(line, comment) {
+                    let commentLine = []
+                    Array.prototype.push.apply(commentLine, line)
+                    
+                    addResource(commentLine, comment);
+
+                    commentLine.push(comment.attributes.text);
+                    if (comment.collection.replyTo) {
+                        commentLine.push(comment.collection.replyTo.id);
+                    } else {
+                        commentLine.push("");
+                    }
+
+                    bookData.push(commentLine);
+                }
+
+                function comment_replies(line, replies) {
+                    _.each(replies, function (comment) {
+                        addCommentLine(line, comment);
+
+                        comment_replies(line, comment.attributes.replies)
+                    });
+                }
```

Comment:
Now that the export functionality is no longer dependent on whether or not we are running with the backend or in "`localStorage`-mode", I think these functions don't belong here, anymore. I would rather see them in `annotation-tool.js`.